tools/lint.mjs

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env node
+
+// Requires Node.js and:
+// * npm install node-html-parser
+//
+// Run script from top level of spec repo after building spec.
+// Example: bikeshed spec && tools/lint.mjs
+//
+// Note that the '.mjs' extension is necessary for Node.js to treat the file as
+// a module. There is an `--experimental-default-type=module` flag but
+// specifying that in the #! line requires trickery that confuses some editors.
+
+
+'use strict';
+import fs from 'node:fs/promises';
+import {parse} from 'node-html-parser';
+
+function log(string) {
+  // console.log(string);
```

Comment:
Should we enable logging by a verbose mode? I am also fine with enabling logging by default.

Reply:
Added `--verbose` option in e520bd0